### PR TITLE
Add travis support

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

**Comment by lulf** [comment]
@grs Please review

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - if [ "$TRAVIS_BRANCH" == "master" ]; then
+      docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS;
+      docker tag $REPO:$COMMIT $REPO:$TAG;
+      docker tag $REPO:$COMMIT $REPO:travis-$TRAVIS_BUILD_NUMBER;
+      docker push $REPO;
+      fi
+
+env:
+    global:
+        - secure: "C83DXTaDhdvtSxrwpHnYVXoL6LeZWWv6vtY892HZ+aWsd0drULPDh1iExwyEE5uEGYpAmKlTYy+ep05+qNpvIkE7JjTmGf0Vs1tkeu1mR2ozhvqZPamokeDQ1eEjc0/Lb1mkVptpUyJrNgllR+o+W+s1dj2IH8PVkE0MK8hjd66e/O1MkfIiUh/kKUoobKeyB6ttQupyqUrGK8vHn48uUOlpoDLCN/Hst0c720yTorMWqb41jaLuKo/NjTZuO3DQ/PTbr1t6Ir1h3kVlzUA1+gQclA3jMcvf+nY0rIEET7Uu9UFfmfTrQao9BkA9QPDWwG7f5BvmGGF8M35DMvbXgA628F2yYBisophJFAu9TiBnD1PgziPoBPdHGqaHDaGNoIkDaq3+cq1czVSWT2wNEaNxdazudOY4SU0p1mGW4iOn5pQ/3saSazkpxOC6DZmZ2SNMAzhexyZomxj1AseTq/sCMKp7hbF0bu2OqVWZH2fCTUdOjn57skmmQa73iVBF1vbIThX127EwoURvg7pMGBBfGImLQ1/fXI6AdgZLLLxabjR28LZYmTHAGcT5mePd6JLX+3Drugb8sZMxr2n+PeRTlet0KLahaBOh1A8XEP3Su1VKgrFc5KgVSUpTQhLv/X4JIrNCLZFmtuVRSy3e34CnCLwNyapQ45TIj7YIBvw=" # DOCKER_EMAIL
```
Comment (grs) [inline .travis.yml]: Just out of curiousity, what are these tokens and where do they come from?

Comment (lulf) [inline .travis.yml]: Its credentials for accessing docker hub.

Comment (lulf) [inline .travis.yml]: ideally, I would prefer if I could just use something like Oauth to authenticate travis for enmasse on docker hub, but I've not found a way of doing that

**Pull request merged**